cfme/tests/infrastructure/test_customization_template.py

```diff
@@ -1,101 +1,105 @@
 # -*- coding: utf-8 -*-
 import fauxfactory
 import pytest
-
-from cfme.infrastructure import pxe
-from cfme.utils import error
-from cfme.utils.blockers import BZ
+from cfme.utils import error, version
```

Comment:
**REQURIED**
Please follow style guides. Put one blank line between external and internal imports.